### ISSUE TITLE
Dynamically add new field to list

### ISSUE THREAD

**Issue by Jordy-A** [kevinchappell/formBuilder#591]
Is it possible to dynamically add fields to the control panel at the right? I have a few custom made elements stored in a database. After initializing formBuilder I would like to call my api, get the elements and dynamically add those elements to the formBuilder panel.

**Comment by kevinchappell** [comment]
No but this would be a good feature.

**Comment by Jordy-A** [comment]
Yes, I would love to see this. It's quite important for my project :)
